### PR TITLE
chore: simpler pull request template

### PR DESCRIPTION
**Pull request by stringhandler**
Simpler PR template

### CODE REVIEW

**Review by delta1 — APPROVED** [review]
LGTM

**Review by sdbondi — APPROVED** [review]
Agree with this. Feels like a more complete checklist should go in https://github.com/tari-project/tari/blob/development/Contributing.md

**Comment by stringhandler** [comment]
Yeah I think the checklist should be handled by CI

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +1,9 @@
-<!--- Provide a general summary of your changes in the Title above -->
+Description
```
Comment (delta1) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
Description:
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
+---
 
-## Description
-<!--- Describe your changes in detail -->
+Motivation and Context
```
Comment (delta1) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
Motivation and Context:
```

**Review by delta1 — APPROVED** [review]

**Pull request merged**